druid-shell/Cargo.toml

```diff
@@ -16,19 +16,20 @@ use_gtk = ["gtk", "gtk-sys", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "cairo
 default-target = "x86_64-pc-windows-msvc"
 
 [dependencies]
-piet-common = "0.0.9"
+piet-common = "0.0.10"
```

Comment:
Let's update piet to 0.0.11 instead after [piet#150](https://github.com/linebender/piet/pull/150) is merged.